COURSE.md

```diff
@@ -101,10 +128,10 @@ This course is divided into separate modules, detailed below. Module A1-A4 are i
 **Exercises**
 *TBD*
 
-##### A4 - Traits and generics
+#### A4 - Traits and generics
 - **Traits**
 - **Commonly used traits from std**
-    - Into/From/TryFrom/TryInto
+    - Into/From/TryFrom/TryInto (referring to A1)
```

Comment:
Eq, PartialEq and Hash may be important too (and the differences between Eq and PartialEq)